client/jetpack-cloud/sections/agency-dashboard/downtime-monitoring/notification-settings/index.tsx

```diff
@@ -206,7 +207,18 @@ export default function NotificationSettings( {
 			<div className="notification-settings__sub-title">{ getSiteCountText( sites ) }</div>
 
 			<form onSubmit={ onSave }>
-				<div className="notification-settings__content">
+				{ isBulkUpdate && (
+					<AlertBanner type="warning">
+						{ translate( 'Settings for selected sites will be overwritten.' ) }
+					</AlertBanner>
+				) }
+				<div
+					className={
+						isBulkUpdate
+							? 'notification-settings__content-with-alert'
+							: 'notification-settings__content'
+					}
+				>
```

Comment:
I see we are adding a class name that does nothing. How about we use `classNames` here?

import classNames from 'classnames';

```suggestion
<div className={ classNames( { 'notification-settings__content': ! isBulkUpdate } ) }>
```